src/overlays/actors/ovl_En_Syateki_Dekunuts/z_en_syateki_dekunuts.c

```diff
@@ -97,46 +112,47 @@ void EnSyatekiDekunuts_Init(Actor* thisx, PlayState* play2) {
     Collider_InitCylinder(play, &this->collider);
     Collider_SetCylinder(play, &this->collider, &this->actor, &sCylinderInit);
 
-    if (EN_SYATEKI_DEKUNUTS_GET_PARAM_F(&this->actor) == 1) {
+    if (EN_SYATEKI_DEKUNUTS_GET_TYPE(&this->actor) == EN_SYATEKI_DEKUNUTS_TYPE_BONUS) {
         Actor_SetScale(&this->actor, 0.01f);
-        this->collider.dim = D_80A2CADC[0];
-        phi_v0 = 3;
+        this->collider.dim = sBonusDekuScrubColliderDimensions[0];
+        unkPathComparison = 3;
     } else {
         Actor_SetScale(&this->actor, 0.02f);
-        phi_v0 = 1;
+        unkPathComparison = 1;
     }
 
-    while (path->unk2 != phi_v0) {
+    while (path->unk2 != unkPathComparison) {
```

Comment:
More so just mentioning. The only other place I've seen unk2 for paths is SubS where it is used in a bitwise flag structure to store day information. Overall it seems to store additional miscellaneous info that actors/code can use to make decisions.